inference-engine/src/low_precision_transformations/src/fake_quantize_dequantization.cpp

```diff
@@ -155,6 +155,26 @@ bool FakeQuantizeDequantization::checkElementwise(const std::shared_ptr<ngraph::
     return true;
 }
 
+std::shared_ptr<Node> FakeQuantizeDequantization::copyWithNewInput(const std::shared_ptr<Node>& input) const {
+    auto lastNode = input;
+    if (convert) {
+        lastNode = convert->copy_with_new_inputs({lastNode});
+    }
+    if (subtract) {
```

Comment:
better `else if`

Reply:
thanks for comment, not an issue: both (`convert` and `subtract`) can exist